package-lock.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Could you please install this dependency with `yarn` instead of `npm`, so that the `yarn.lock` file is used?

Reply:
done